Hazel/src/Hazel/Scene/Scene.cpp

```diff
@@ -94,7 +94,7 @@ namespace Hazel {
 		{
 			auto [transform, sprite] = group.get<TransformComponent, SpriteRendererComponent>(entity);
 
-			Renderer2D::DrawQuad(transform.GetTransform(), sprite.Color);
+			Renderer2D::DrawQuad(transform.GetTransform(), sprite.Texture, sprite.TilingFactor, sprite.Color);
```

Comment:
I think you have to update this, as there is mouse-picking in Hazel. The entity ID is missing here 🙂